pkg/discoveryservice/manager.go

```diff
@@ -92,7 +91,8 @@ func (dsm *Manager) Start(ctx context.Context) {
 			ClientAuth:   tls.RequireAndVerifyClientCert,
 			ClientCAs:    loadCA(dsm.CACertificatePath, setupLog),
 		},
-		rollback.OnError(mgr.GetClient()),
+		// rollback.OnError(mgr.GetClient()),
```

Comment:
is this comment on purpose?

Reply:
Yes, is the temporary disable of the self-healing functionality.

Reply:
nice!